libkbfs/folder_branch_ops.go

```diff
@@ -4704,6 +4704,11 @@ func (fbo *folderBranchOps) applyMDUpdatesLocked(ctx context.Context,
 				return err
 			}
 		}
+		if rmd.IsRekeySet() {
```

Comment:
Thanks for checking strib! Although I'm still not sure I understand the concern. If a MD update with `rekey=true` comes in, it generates a rekey request, and the rekey FSM doesn't do anything other than printing a log if the paper prompt is set: https://github.com/keybase/kbfs/blob/1faa48ab5973de7974186b1b84e71c0026921a56/libkbfs/rekey_fsm.go#L278-L292